zc.recipe.egg_/CHANGES.rst

```diff
@@ -4,7 +4,7 @@ Change History
 3.0.0 (unreleased)
 ==================
 
-- Drop support for Python 2.  Require Python 3.8 as minimum.
+- Drop support for Python 2.  Require Python 3.9 as minimum.
   Require ``zc.buildout`` 3.1.0 as minimum.
```

Comment:
Should we also have a release of this subpackage?

Reply:
It could be good, though it seems not strictly necessary.  But we are currently not actually running the tests of `zc.recipe.egg`.  And `bin/oltest` which *does* run these tests (plus the buildout ones) fails locally.  So there is some work to do still before we can really release that one.